test_botframework.py

```diff
@@ -0,0 +1,76 @@
+from errbot import Message
+
+from botframework import BotFramework, Identifier, Conversation
+from test_common import mock_config
+from unittest.mock import MagicMock, call, patch
+import pytest
+
+member_id = 1
+member_name = 'member name'
+member_email = 'member@email.com'
+conversation_id = 1
+default_reply = {
+    'type': 'message',
+    'text': 'My text message'
+}
+
+class Test_send_message:
```

Comment:
since you are using `pytest`, could you please remove `class` and move test to module level?

Reply:
We can definitively remove classes, but we usually prefer to use groupings (tests suites), as you would use with any BDD framework (e.g. [Jasmine](https://jasmine.github.io/)). So the class, is just a shortcut for wrapping all the tests related to a method. Without classes, you would need to have very long names for your methods and the file tends to become hard to read.